script/hassfest/requirements.py

```diff
@@ -22,7 +22,7 @@
     commented.lower().replace("_", "-") for commented in COMMENT_REQUIREMENTS
 }
 PACKAGE_REGEX = re.compile(
-    r"^(?:--.+\s)?([-_\.\w\d\[\]]+)(==|>=|<=|~=|!=|<|>|===)*(.*)$"
+    r"^(?:--.+\s)?([-_,\.\w\d\[\]]+)(==|>=|<=|~=|!=|<|>|===)*(.*)$"
```

Comment:
I added the comma to the package name as the previous regex didn't allow multiple extras.